server.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const connectDB = require("./config/db");
+const test = require("./routes/api/test");
+
+const app = express();
+
+// Init Middleware
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+//Connect Database
+connectDB();
+
+//Routes
+app.use("/api/test", test);
```

Comment:
I believe we should add some opinionated folder structure for routes and controllers models etc.